dev-docs/bidders/freestar.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: bidder
+title: Freestar
+description: Prebid Freestar Bidder Adaptor
+biddercode: freestar
+hide: true
+pbjs: true
+media_types: banner
+gdpr_supported: true
+usp_supported: true
+schain_supported: true
+prebid_member: true
+---
+
+**Table of Contents**
+
+- [Bid params](#freestar-bid-params)
+- [Banner Ads](#freestar-banner)
+- [Configuration](#freestar-configuration)
+
+<a name="freestar-bid-params" />
+
+### Bid params
+
+{: .table .table-bordered .table-striped }
+| Name | Scope | Description | Example |
+|------+-------+-------------+---------|
+| `tagId` | required | The tag ID from Freestar. | `"test-pb-leaderboard"` |
+| `siteId` | optional | Override the global Freestar site ID, [see here](#freestar-configuration). Please reach out to your Freestar Account Manager for more details. | `"example.com"` |
+| `floor` | optional | The minumum CPM (in USD) requred to participate | `0.25` |
```

Comment:
please mention you support the floors module and if this overrides the module or is a fallback